tests/snuba/test_organization_discover_query.py

```diff
@@ -244,16 +245,16 @@ def test_zerofilled_dates_when_rollup_absolute(self):
                 'fields': ['project.name'],
                 'groupby': ['time'],
                 'orderby': '-time',
-                'start': (datetime.now() - timedelta(seconds=60)).strftime('%Y-%m-%dT%H:%M:%S'),
-                'end': (datetime.now()).strftime('%Y-%m-%dT%H:%M:%S'),
-                'rollup': 10,
+                'start': (self.now - timedelta(seconds=300)).strftime('%Y-%m-%dT%H:%M:%S'),
```

Comment:
Seems possible that previously, the event that was created could potentially move into a different time bucket. Now the start/end times are based on when the event is created.